diff --git a/dist/commands/test.js b/dist/commands/test.js
index 179ec3b0b6826e215a6aad3e90dcee4042b1ab2f..a3014a9785550242c5be9165c45f7c40cdd2fbb1 100644
--- a/dist/commands/test.js
+++ b/dist/commands/test.js
@@ -252,7 +252,7 @@ async function runDocker(datasource, opts) {
         // Build the `docker run` command options and flags
         const dockerRunOpts = [
             'run',
-            '-it',
+            // '-it',
             '--rm',
             '--mount',
             `type=bind,source=${current_folder},target=/matchstick`,
@@ -278,14 +278,18 @@ async function runDocker(datasource, opts) {
                 stdio: 'inherit',
             }).on('close', code => {
                 if (code === 0) {
-                    spawn('docker', dockerRunOpts, { stdio: 'inherit' });
+                    spawn('docker', dockerRunOpts, { stdio: 'inherit' }).on('close', (code) => {
+                        if (code !== 0) process.exit(code);
+                    });
                 }
             });
         }
         else {
             this.log('Docker image already exists. Skipping `docker build` command...');
             // Run the container from the existing matchstick docker image
-            spawn('docker', dockerRunOpts, { stdio: 'inherit' });
+            spawn('docker', dockerRunOpts, { stdio: 'inherit' }).on('close', (code) => {
+                if (code !== 0) process.exit(code);
+            });
         }
     });
 }
