diff --git a/node_modules/@nomiclabs/hardhat-solhint/dist/src/index.js b/node_modules/@nomiclabs/hardhat-solhint/dist/src/index.js
index acf1244..cbaff27 100644
--- a/node_modules/@nomiclabs/hardhat-solhint/dist/src/index.js
+++ b/node_modules/@nomiclabs/hardhat-solhint/dist/src/index.js
@@ -30,8 +30,7 @@ function getDefaultConfig() {
 }
 function getFormatter(formatterName = "stylish") {
     try {
-        const solhintPath = require.resolve("solhint");
-        const formatterPath = require.resolve(`eslint/lib/formatters/${formatterName}`, { paths: [solhintPath] });
+        const formatterPath = require.resolve(`solhint/lib/formatters/${formatterName}`);
         return require(formatterPath);
     }
     catch (ex) {
