diff --git a/node_modules/nx/src/tasks-runner/task-env.js b/node_modules/nx/src/tasks-runner/task-env.js
index 7c80c20..276eab5 100644
--- a/node_modules/nx/src/tasks-runner/task-env.js
+++ b/node_modules/nx/src/tasks-runner/task-env.js
@@ -16,7 +16,7 @@ exports.getEnvVariablesForBatchProcess = getEnvVariablesForBatchProcess;
 function getTaskSpecificEnv(task) {
     // Unload any dot env files at the root of the workspace that were loaded on init of Nx.
     const taskEnv = unloadDotEnvFiles({ ...process.env });
-    return process.env.NX_LOAD_DOT_ENV_FILES === 'true'
+    return false === 'true'
         ? loadDotEnvFilesForTask(task, taskEnv)
         : // If not loading dot env files, ensure env vars created by system are still loaded
             taskEnv;
