diff --git a/node_modules/nx/src/tasks-runner/forked-process-task-runner.js b/node_modules/nx/src/tasks-runner/forked-process-task-runner.js
index 5013ad9..cbd14ef 100644
--- a/node_modules/nx/src/tasks-runner/forked-process-task-runner.js
+++ b/node_modules/nx/src/tasks-runner/forked-process-task-runner.js
@@ -177,7 +177,12 @@ class ForkedProcessTaskRunner {
     }
     // region Environment Variables
     getEnvVariablesForProcess() {
-        return Object.assign(Object.assign(Object.assign({}, this.getDotenvVariablesForForkedProcess()), process.env), this.getNxEnvVariablesForForkedProcess(process.env.FORCE_COLOR === undefined ? 'true' : process.env.FORCE_COLOR));
+      // This patch disables default nx environment variables loading to keep existing order specified by NestJS services.
+        return Object.assign(
+          {},
+          // Object.assign(Object.assign({}, this.getDotenvVariablesForForkedProcess()), process.env),
+          this.getNxEnvVariablesForForkedProcess(process.env.FORCE_COLOR === undefined ? 'true' : process.env.FORCE_COLOR)
+        );
     }
     getEnvVariablesForTask(task, forceColor, outputPath, streamOutput) {
         const res = Object.assign(Object.assign(Object.assign({}, this.getDotenvVariablesForTask(task)), process.env), this.getNxEnvVariablesForTask(task, forceColor, outputPath, streamOutput));
